gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
```

Do we need `all`? I tend to use `bin` for the small download size.

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```